test/capture.js

```diff
@@ -1,3 +1,21 @@
+define([ 'intern!tdd',
+        'intern/chai!expect',
+        '../pointerevents',
+        'src/boot',
+        'src/pointermap',
+        'src/capture',
+        'src/dispatcher',
+        'src/installer',
+        'src/mouse',
+        'src/ms',
+        'src/platform-events',
+        'src/PointerEvent',
+        'src/touch',
+        'src/touch-action'
+       ],
+       function (tdd, expect) {
+           with (tdd) {
```

Comment:
Whatever is being used from `tdd` should just be explicit, for example if `suite()` is coming from `tdd`, then it becomes `tdd.suite()`.

I'm surprised this is in the Intern docs since it's forbidden in strict mode.